src/LtiAssignmentsGradesService.php

```diff
@@ -116,6 +116,20 @@ public function getLineItems(): array
         return $lineitems;
     }
 
+    public function getLineItem(string $url): LtiLineitem
+    {
+        if (!in_array(LtiConstants::AGS_SCOPE_LINEITEM, $this->getScope())) {
+            throw new LtiException('Missing required scope', 1);
+        }
+
+        $request = new ServiceRequest(LtiServiceConnector::METHOD_GET, $url);
+        $request->setAccept(static::CONTENTTYPE_LINEITEM);
+
+        $lineItem = $this->get($request);
```

Comment:
We should get rid of this and just use `makeServiceRequest()` for the GET request, then use the response to instantiate the LtiLineItem in this method.